src/Components/PagesBody/LandingPage/LandingPage.tsx

```diff
@@ -5,23 +5,43 @@ import ResponsiveGrid from '../../Template/Layout/ResponsiveGrid';
 const serviceDataMock = [
```

Comment:
We have that data in DB, will be good to change to real fetch. Also could be as TODO, not in this task